web/priv/js_effects/word_game.js

```diff
@@ -0,0 +1,3151 @@
+            "golem",
+            "loopy",
+            "round",
+            "audit",
+            "lying",
+            "gamma",
+            "labor",
+            "islet",
+            "civic",
+            "forge",
+            "corny",
+            "moult",
+            "basic",
+            "salad",
+            "agate",
+            "spicy",
+            "spray",
+            "essay",
+            "fjord",
+            "spend",
+            "kebab",
+            "guild",
+            "aback",
+            "motor",
+            "alone",
+            "hatch",
+            "hyper",
+            "thumb",
+            "thumb",
+            "ought",
+            "ought",
+            "dutch",
+            "pilot",
+            "tweed",
+            "comet",
+            "jaunt",
+            "jaunt",
+            "steed",
+            "abyss",
+            "growl",
+            "fling",
+            "dozen",
+            "dozen",
+            "erode",
+            "world",
+            "gouge",
+            "click",
+            "briar",
+            "great",
+            "altar",
+            "pulpy",
+            "blurt",
+            "coast",
+            "duchy",
+            "duchy",
+            "fixer",
+            "group",
+            "rogue",
+            "badly",
+            "smart",
+            "pithy",
+            "gaudy",
+            "chill",
+            "heron",
+            "heron",
+            "finer",
+            "surer",
+            "radio",
+            "rouge",
+            "perch",
+            "perch",
+            "wrote",
+            "clock",
+            "tilde",
+            "store",
+            "prove",
+            "bring",
+            "solve",
+            "cheat",
+            "grime",
+            "exult",
+            "usher",
+            "epoch",
+            "triad",
+            "break",
+            "rhino",
+            "viral",
+            "conic",
+            "masse",
+            "sonic",
+            "vital",
+            "trace",
+            "using",
+            "peach",
+            "champ",
+            "baton",
+            "brake",
+            "pluck",
+            "craze",
+            "gripe",
+            "weary",
+            "picky",
+            "acute",
+            "ferry",
+            "aside",
+            "tapir",
+            "troll",
+            "unify",
+            "rebus",
+            "boost",
+            "truss",
+            "siege",
+            "tiger",
+            "tiger",
+            "slump",
+            "crank",
+            "gorge",
+            "query",
+            "drink",
+            "favor",
+            "abbey",
+            "tangy",
+            "panic",
+            "solar",
+            "shire",
+            "proxy",
+            "point",
+            "robot",
+            "robot",
+            "wince",
+            "crimp",
+            "knoll",
+            "sugar",
+            "whack",
+            "whack",
+            "whack",
```

Comment:
just curious - why so many duplicated words? Just to increase probability of them for some reason??